README.md

```diff
@@ -12,6 +12,7 @@ Playbook is the first design system built for both Rails & React interfaces. Ins
 
 ## Getting started
 
+1. `cd playbook`
```

Comment:
@thestephenmarshall @jasperfurniss hello! I added this here as it got me stuck for a while not realising which folder I had to be in. Is this okay?

Reply:
All of this is about to change with Carlo's refactor anyways. But yes. Thanks for updating.